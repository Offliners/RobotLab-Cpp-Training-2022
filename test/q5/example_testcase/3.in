1+2=*5+(1+7+11)=%5-11=