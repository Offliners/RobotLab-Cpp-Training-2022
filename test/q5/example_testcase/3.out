3
34
-7